functions-app/school-import/readme.md

```diff
@@ -0,0 +1,14 @@
+# BlobTrigger - JavaScript
+
+The `BlobTrigger` makes it incredibly easy to react to new Blobs inside of Azure Blob Storage. This sample demonstrates 

```

Comment:
MS Marketing blurb alert 🔔

Reply:
@GuyHarwood Good spot!  I've updated the readme with something more useful. ⚔️